app/ui/legacy/src/main/java/com/fsck/k9/activity/MessageCompose.java

```diff
@@ -363,7 +364,25 @@ public void onTextChanged(CharSequence s, int start, int before, int count) {
 
         subjectView.addTextChangedListener(draftNeedsChangingTextWatcher);
 
-        messageContentView.addTextChangedListener(draftNeedsChangingTextWatcher);
```

Comment:
There's no need to remove this. Add an additional `TextWatcher` that only deals with detecting pasted URIs.